editor-components/feature/feature9/feature9.module.scss

```diff
@@ -62,7 +62,7 @@
 .sub-title-content{
   width: 50%;
   padding-right: 20%;
-  color: var(   --text-color-white) ;//white;
+  color: var(   --text-color-white) ;
```

Comment:
i guess this is not the correct color variables. maybe it should be var(--composer-font-color-primary).
please check kardelen's discord message in general channel.